### ISSUE TITLE
Upgrading from 1.0 to 2.0.alpha* causes severe lag

### ISSUE THREAD

**Issue by orecus** [hyperion-project/hyperion.ng#879]
- [x] I confirm that this is an issue rather than a question.

## Bug report
I recently tried to update Hyperion.NG running on my Vero 4k+ from 1.0 (380f9b5d) to 2.0.alpha.5 (e796155) and 2.0.alpha.6 (f5583de) but the newer versiones introduced severe lagging of both effects and grabbing on my device. 

I have not tried the earlier versions of 2.0, just the two latest, but if it would be useful I can test those too and see if earlier versions are better compared to the newer ones, I've also not tested the latest commit.

I have a NodeMCU attached trough WiFi which controls the LED's (277 LEDs) and Hyperion.NG connects to this trough the UDPRAW configuration. The NodeMCU is running WLED 0.10 and any animations set by it works fine, nothing is changed on the NodeMCU side between the upgrade of Hyperion.NG.

If I downgrade (Uninstall -> Install) 380f9b5d it's back to be butter smooth in both grabbing and effects with no lagging effects.

#### Steps to reproduce
Upgrade to any of the mentioned new versions.

#### What is expected?
That effects and grabbing is butter smooth.

#### What is actually happening?
Both effects and grabbing introduces severe lagging.

#### System
Hyperion Server: 
- Build:       (HEAD detached at f5583de8) (Sersch-f7865b45/f5583de8-1590611619)
- Build time:  Jul 16 2020 09:05:52
- Git Remote:  https://github.com/hyperion-project/hyperion.ng.git
- Version:     2.0.0-alpha.6
- UI Lang:     auto (BrowserLang: sv-SE)
- UI Access:   expert
- Avail Capt:  v4l2,framebuffer,amlogic,qt

Hyperion Server OS: 
- Distribution: Open Source Media Center
- Arch:         arm64
- Kernel:       linux (3.14.29-158-osmc (WS: 32))
- Browser:      Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36

**Comment by tpmodding** [comment]
we have a loooot of changes with the next release, i would recommend to wait for next release before we investigate too much time, if the problems still occure then we can have a deeper look, is this okay for you?

**Comment by orecus** [comment]
@tpmodding I think I have narrowed it down, I continued yesterday and tested a few things. What I did do that got it all to work as expected, was that I removed all previous configuration and did a full uninstall.

Then I installed the latest commit in the repository and configured everything from scratch, then it all worked perfectly. I'm not sure if it's something fixed in the latest version compared to the released alpha.6, or if there perhaps is something in the configuration that have changed between 1.0 -> 2.0 that can cause it to behave like that? 

One thing that stood out to me, is that in 1.0 I had Latch Time set and it caused no problems (I think I used some default value possibly) but in 2.0 it's 0. Not sure if the behaviour of that have changed? It was the only thing that stood out too me when I compared the configuration for the working 2.0 version, and my old backup of 1.0.
